velox/row/tests/UnsafeRowBatchDeserializerTest.cpp

```diff
@@ -716,17 +716,6 @@ TEST_F(UnsafeRowComplexBatchDeserializerTests, nullRows) {
     const auto& innerRowVector =
         createInputRow(innerBatchSize, [](vector_size_t i) { return true; });
     const auto& outerRowVector = makeRowVector({innerRowVector});
-
-    auto rowSize = UnsafeRowDynamicSerializer::serialize(
-        outerRowVector->type(), outerRowVector, buffers_[0], 0);
-    ASSERT_TRUE(rowSize.has_value());
-    serializedVector.push_back(std::string_view(buffers_[0], rowSize.value()));
-    VectorPtr outputVector =
-        UnsafeRowDynamicVectorBatchDeserializer::deserializeComplex(
-            serializedVector, outerRowVector->type(), pool_.get());
-    assertEqualVectors(outerRowVector, outputVector);
+    testVectorSerde(outerRowVector);
```

Comment:
Is the TODO above still valid?

```
// TODO: The serde is still buggy as tests with innerBatchSize larger than 1
  // is currently still failing.
```

Reply:
Yeah it is still valid. This is the bug we need to fix after refactoring